src/components/authorize/tests/authorize-controller.test.ts

```diff
@@ -41,8 +41,12 @@ describe("authorize controller", () => {
     mockClaims = createmockclaims();
     req = mockRequest({
       path: PATH_NAMES.AUTHORIZE,
-      session: { client: {}, user: {} },
-      log: { info: sinon.fake() },
+      session: {
+        client: {},
+        user: {},
+        save: (callback: () => void) => callback(),
+      },
+      log: { info: sinon.fake(), debug: sinon.fake() },
```

Comment:
I wonder if these mocked properties could be extracted and reused, as they are seemingly quite common in these tests